components/datetime/src/options/preferences.rs

```diff
@@ -42,6 +42,7 @@ use serde::{Deserialize, Serialize};
 /// ```
 #[derive(Debug, Clone, PartialEq, Copy)]
 #[cfg_attr(feature = "serialize", derive(Serialize, Deserialize))]
+#[allow(clippy::exhaustive_structs)] // this type is stable
 pub struct Bag {
```

Comment:
Question: How do we know this is stable?

Reply:
ah, it shouldn't be

Reply:
(fixing)